sdk/identity/identity/src/credentials/azureCliCredential.ts

```diff
@@ -158,13 +159,9 @@ export class AzureCliCredential implements TokenCredential {
         }
         try {
           const responseData = obj.stdout;
-          const response: { accessToken: string; expiresOn: string } = JSON.parse(responseData);
+          const response: AccessToken = this.parseRawResponse(responseData);
           logger.getToken.info(formatSuccess(scopes));
-          const returnValue = {
-            token: response.accessToken,
-            expiresOnTimestamp: new Date(response.expiresOn).getTime(),
-          };
-          return returnValue;
+          return response;
```

Comment:
nit: simplify to just `return this.parseRawResponse(obj.stdout)`?

Reply:
It would be a good simplification for sure but I want to ensure we can properly parse the response before logging success on line 163, so for now I opted to leave as is